### ISSUE TITLE
Jupyter notebook password

### ISSUE THREAD

**Issue by psychemedia** [mmh352/ou-container-builder#47]
It would be useful to be able to specify a password/token used to access the Jupyter notebook container.

**Comment by mmh352** [comment]
The security implications of having a static password need to be considered. Possibly something to only enable if the container is run locally, rather than behind a JupyterHub instance.

**Comment by psychemedia** [comment]
Agreed. But the default could be to have the token generated route. You can pass a token in as an env var when you start a local container but it's just more fiddliness when testing various things...

Actually - can you set env vars via the config script? That would be neater.
